### PR TITLE
Collect image, size, location and platform info when deploy failed.

### PR DESCRIPTION
**Pull request by LiliDeng**
Even deploy fail, we still need collect as much info as we can.

### CODE REVIEW

**Review by squirrelsc — APPROVED** [review]

File: lisa/node.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def __init__(
         self.capability = capability
         self.name: str = ""
         self.index = index
+        self.is_ready: bool = False
```
Comment (squirrelsc) [inline lisa/node.py]: Change it's name from `is_ready` to `is_connected` to make it more accurate and same as new environment status.

Comment (squirrelsc) [reply lisa/node.py]: And reset it to False in `close`.

**Review by squirrelsc — CHANGES_REQUESTED** [review]
Provided some missed comments

**Review by squirrelsc — COMMENTED** [review]
Merge latest commits to previous, because it handles same things in this PR.

**Comment by LiliDeng** [comment]
> Merge latest commits to previous, because it handles same things in this PR.

Done.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**